README.md

```diff
@@ -13,4 +13,4 @@ and use it 3 times inside the `App` instead of static markup.
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_person/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://github.com/dmytro-voronin/react_person/) and add it to the PR description.
```

Comment:
Consider changing this in this way `(https://dmytro-voronin.github.io/react_person/)`  `https://<your_account>.github.io/react_person/` is required structure for workable link